api/src/main/java/org/openmrs/module/fhir2/api/impl/BaseFhirService.java

```diff
@@ -153,8 +154,12 @@ public T patch(@Nonnull String uuid, @Nonnull PatchTypeEnum patchType, @Nonnull
 					updatedFhirObject = JsonPatchUtils.applyJsonPatch(fhirContext, existingFhirObject, body);
 				}
 				break;
+			case XML_PATCH:
+				updatedFhirObject = XmlPatchUtils.applyXmlPatch(fhirContext, existingFhirObject, body);
+				break;
 			default:
```

Comment:
We don't need the `default` case anymore do we?

Reply:
done!